kubernetes/provider_test.go

```diff
@@ -9,6 +10,9 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/hashicorp/terraform/terraform"
 	"github.com/terraform-providers/terraform-provider-google/google"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	api "k8s.io/kubernetes/pkg/api/v1"
```

Comment:
🤔 would it make sense to name this `apiv1` for consistency?

Reply:
It was actually intention to make it version-agnostic, so that in case the future API versions are same, or similar we only need to update the import path without cluttering the diff too much.